app/scripts/controllers/landing.js

```diff
@@ -44,10 +45,26 @@ angular.module('confRegistrationWebApp')
         };
         $scope.searchComplete = val;
         $scope.searchResults = response.data;
+
+        $scope.filterUpdate();
       });
     }, 500, {leading: false});
 
-    $scope.dateFilter = function(event){
+    $scope.filterSearchResults = function(){
+      return _.filter($scope.searchResults, event => {
+        return dateFilter(event) && locationFilter(event);
+      });
+    };
+
+    $scope.filterUpdate = function(){
+      analytics.digitalData.searchTerm = $scope.searchVal;
+      analytics.digitalData.searchLocationFilter = $scope.eventFilters.locationName;
+      analytics.digitalData.searchDateFilter = $scope.eventFilters.date;
```

Comment:
SSO GUID and GR ID are not provided by the API so I can't hand them to you at the moment. I could tell you if the user signed in with Facebook or Relay if that is useful but right now I don't have any IDs on the front end.

Page titles still need work. That will probably happen in another pull request. Are you ok with this going live without page titles or should we wait? The page titles are currently defined in the wrong place to be useful for analytics.

The search direct call rule should be named `search`and should fire anytime seach parameters are changed but should not fire a `page view` https://github.com/CruGlobal/conf-registration-web/pull/544/files#diff-78cedf0c6c09b9633fef47656a98872aR64. If you are seeing both I guess I need to do some debugging.